internal/backup/collector_system.go

```
@@ -787,163 +797,211 @@ func (c *Collector) collectSystemCommands(ctx context.Context) error {
+			"zfs get all",
+			filepath.Join(zfsDir, "zfs_get_all.txt"),
+			"ZFS properties",
+		)
+	}
+
 	return nil
 }
 
+func (c *Collector) collectSystemLVMRuntime(ctx context.Context, commandsDir string) error {
+	if _, err := c.depStat(c.systemPath("/sbin/pvs")); err != nil {
+		return nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 979) but it returns nil

(nilerr)